### PR TITLE
Display warning message that AzureRM will retire

### PR DESCRIPTION
**Pull request by erich-wang**
Two changes in this RP:
1. Display warning message that AzureRM will retire
2. Add missing pieces for  official 1.1.8 version

### CODE REVIEW

File: src/Common/AzurePSCmdlet.cs [hunk 1]
```diff
@@ -365,6 +366,32 @@ protected override void BeginProcessing()
...
+            bool supressRetireWarningMessage = false;
+
+            try
+            {
+                supressRetireWarningMessage = bool.Parse(Environment.GetEnvironmentVariable("SuppressAzureRmModulesRetiringWarning"));
+            }
+            catch (Exception)
+            {
+                //no action
+            }
```
Comment (BethanyZhou) [inline src/Common/AzurePSCmdlet.cs:387]: How about using tryParse()
bool.TryParse(Environment.GetEnvironmentVariable("SuppressAzureRmModulesRetiringWarning"), out supressRetireWarningMessage);

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
LGTM, left one message.

File: src/Common/AzurePSCmdlet.cs (hunk 1 above)
Comment (erich-wang) [reply src/Common/AzurePSCmdlet.cs:387]: Good suggestion. However Environment.GetEnvironmentVariable("SuppressAzureRmModulesRetiringWarning") may throw exception per https://docs.microsoft.com/en-us/dotnet/api/system.environment.getenvironmentvariable?view=net-5.0, we need to use try catch anyway...

Comment (BethanyZhou) [reply src/Common/AzurePSCmdlet.cs:387]: Make sense. Thank your for clarification.

**Pull request merged**